### PR TITLE
Pass through speech rate.

### LINKED ISSUE #2: Sounds stupid
Super slow - cant seem to change the utterance rate?  Takes like 6 seconds to say a sentence with weird pauses.

### PR DESCRIPTION
**Pull request by tannewt**
Pass through speech rate or use the default.

---
Closes #2

### CODE REVIEW

**Comment by nashpro** [comment]
Hi, could you update the documentation? Thanks!

File: SpeechSynthesizer.m [hunk 1]
```diff
@@ -13,37 +13,39 @@ @implementation SpeechSynthesizer
...
     AVSpeechUtterance *utterance = [[AVSpeechUtterance alloc] initWithString:text];
-    
+
     utterance.voice = [AVSpeechSynthesisVoice voiceWithLanguage:voiceLanguage];
-    
-    // SHOULD BE SET BY ARGUMENT
-    utterance.rate = 0.25;
-    
+
+    if (rate) {
```
Comment (naoufal) [inline SpeechSynthesizer.m]: What happens when no utterance is specified?

**Comment by naoufal** [comment]
@tannewt This change looks good! I just have a question around how the fix handles no utterance rate being passed.  

Once that's cleared up and the docs are updated, we should be good merge this.

**Comment by bradynapier** [comment]
Just add a default rate at the top

**Comment by joshbuchea** [comment]
Is the plan still to merge this?

**Comment by naoufal** [comment]
@joshbuchea Yep, still waiting on @tannewt to update the documentation.

> Once that's cleared up and the docs are updated, we should be good merge this.

**Comment by tannewt** [comment]
Sorry all. I had forgotten about this. :-) I added docs and responded to the question about the default. @naoufal let me know if you have more feedback.

**Comment by naoufal** [comment]
@tannewt :rocket: LGTM, thanks for your contribution!

**Comment by naoufal** [comment]
@nashpro @joshbuchea @tannewt Merged and published on npm. https://www.npmjs.com/package/react-native-speech  Thanks for the help all.

**Comment by tannewt** [comment]
Awesome! No problem.

**Comment by joshbuchea** [comment]
Awesome! Thank you for your efforts gentlemen.

**Pull request merged**